docs/validators/delegation-manager.mdx

```diff
@@ -136,43 +136,84 @@ Setting the identity of the staking pool in the metadata is the **first step** i
+As of January 2024, the only accepted way to customize the information of a delegation contract is via MultiversX Assets.
+Provisioning of information from keybase or your Github repository is no longer accepted and you should migrate to the new approach is you still use those.
 :::
 
-### Display information
-
-To customize the information for your delegation contract, which will be available in the lists displayed on the delegation pages both in xPortal and the web wallet, some additional information has to be added on a GitHub account with the same identifier as the identity. If a GitHub username matching the node identity is already taken, you will need to change the identity in the node provider to a GitHub username that you control.
+To customize the information for your delegation contract, which will be available inside the MultiversX ecosystem (such as Explorer, Web Wallet, xPortal, and so on),
+some additional information has to be added to the MultiversX assets repository.
 
-Please fill in the **avatar picture** and edit the profile providing the **name**, **bio**, **website** and **twitter** (if applicable). This information together with the **service fee, percentage filled** and **APR** will be displayed for every delegation contract on the delegation pages in the web wallet and xPortal. If this information cannot be found a generic logo and the delegation contract's address is displayed.
+In order to do so, the owner of the nodes/staking provider must open a Pull Request against https://github.com/multiversx/mx-assets (`master` branch).
 
-In order to complete the matching between the delegation contract and GitHub identity of the staking pool, a repository needs to be created with the name "multiversx" which contains one single file: "keys.json". This file contains a json array with the provider's smart contract address, just like it was on Keybase. For devnet/testnet, you need to have a folder called devnet/testnet in the same repository, with the "keys.json" file underneath.
+_Step 1: choose the environment_
+- for mainnet go to `mx-assets/identities`
+- for testnet go to `mx-assets/testnet/identities`
+- for devnet go to `mx-assets/devnet/identities`
 
-An example for the contents of the `keys.json` file for the `"testmxprovider"` GitHub identity would be:
+_Step 2: create a directory with an unique name_
+Must be lowercase, alphanumeric, with no spaces. For example, `my-new-identity`
 
+_Step 3: add a `info.json` file_
+Create a `info.json` file where you specify the owned address(es), the name, the description and social links.
+Example:
 ```json
- [
-  "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py"
- ]
+{
+  "description": "This is my new identity",
+  "name": "New identity",
+  "website": "http://newidentity.com",
+  "twitter": "https://twitter.com/newidentity",
+  "location": "France",
+  "owners": [
+    "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz8llllsh6u4jp", // staking provider address
```

Comment:
this is the Arc Stake SP :D

Reply:
only on mainnet. On testnet/devnet it might not even exist. However, will put a dummy sc address